blogposts/2021/dev-tool-time-seth-vargo.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: 'Dev Tool Time with Seth Vargo: Productivity hacks and .gitconfig tips'
 description: Check out the recording of our first episode of Dev Tool Time, in which Google Cloud Engineer Seth Vargo shares his tips for a productivity-optimized desk setup, efficient window management, and keyboard shortcuts.
+seoTitle: Productivity Hacks & .gitconfig Tips – Dev Tool Time
```

Comment:
The name "seoTitle" seems a bit misleading considering that `<title>` also controls the tab title 🤔

Reply:
This might be a compelling reason to have the "seoTitle" just be "title," and explicitly set the visual title that goes on the post page